module/unit.sh

```diff
@@ -456,7 +456,7 @@ function ::unit:test() {
                 module=${module%.sh}
                 if [ ${#g_MODULES[@]} -eq 0 -o ${g_MODULES[${module}]--1} -eq 1 ]; then
```

Comment:
So `${foo:-bar}` means that report `$foo`, but if it is not set, or it is set to nothing then report `$bar`.